{"timestamp": "2024-12-01T14:56:35.492530", "level": "ERROR", "message": "Error in admin_add_property: 1265 (01000): Data truncated for column 'status' at row 1", "module": "admin", "function": "admin_add_property", "line": 447, "thread_id": 129226334742336, "environment": "development", "exception": {"type": "DatabaseError", "message": "1265 (01000): Data truncated for column 'status' at row 1", "traceback": "Traceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 706, in cmd_query\n    self._cmysql.query(\n_mysql_connector.MySQLInterfaceError: Data truncated for column 'status' at row 1\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/routes/admin.py\", line 388, in admin_add_property\n    property_id_result = execute_procedure(\n                         ^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/core/database.py\", line 50, in execute_procedure\n    cursor.callproc(procedure_name, params)\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/cursor_cext.py\", line 566, in callproc\n    result = self._connection.cmd_query(\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/opentelemetry/context_propagation.py\", line 97, in wrapper\n    return method(cnx, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 714, in cmd_query\n    raise get_mysql_exception(\nmysql.connector.errors.DatabaseError: 1265 (01000): Data truncated for column 'status' at row 1"}}
{"timestamp": "2024-12-01T15:01:53.881602", "level": "ERROR", "message": "Error in admin_add_property: 1265 (01000): Data truncated for column 'status' at row 1", "module": "admin", "function": "admin_add_property", "line": 448, "thread_id": 137674083014464, "environment": "development", "exception": {"type": "DatabaseError", "message": "1265 (01000): Data truncated for column 'status' at row 1", "traceback": "Traceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 706, in cmd_query\n    self._cmysql.query(\n_mysql_connector.MySQLInterfaceError: Data truncated for column 'status' at row 1\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/routes/admin.py\", line 389, in admin_add_property\n    property_id_result = execute_procedure(\n                         ^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/core/database.py\", line 50, in execute_procedure\n    cursor.callproc(procedure_name, params)\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/cursor_cext.py\", line 566, in callproc\n    result = self._connection.cmd_query(\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/opentelemetry/context_propagation.py\", line 97, in wrapper\n    return method(cnx, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 714, in cmd_query\n    raise get_mysql_exception(\nmysql.connector.errors.DatabaseError: 1265 (01000): Data truncated for column 'status' at row 1"}}
{"timestamp": "2024-12-01T15:03:01.995591", "level": "ERROR", "message": "Error in admin_add_property: 1265 (01000): Data truncated for column 'p_status' at row 1", "module": "admin", "function": "admin_add_property", "line": 448, "thread_id": 137674083014464, "environment": "development", "exception": {"type": "DatabaseError", "message": "1265 (01000): Data truncated for column 'p_status' at row 1", "traceback": "Traceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 706, in cmd_query\n    self._cmysql.query(\n_mysql_connector.MySQLInterfaceError: Data truncated for column 'p_status' at row 1\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/routes/admin.py\", line 389, in admin_add_property\n    property_id_result = execute_procedure(\n                         ^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/core/database.py\", line 50, in execute_procedure\n    cursor.callproc(procedure_name, params)\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/cursor_cext.py\", line 566, in callproc\n    result = self._connection.cmd_query(\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/opentelemetry/context_propagation.py\", line 97, in wrapper\n    return method(cnx, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 714, in cmd_query\n    raise get_mysql_exception(\nmysql.connector.errors.DatabaseError: 1265 (01000): Data truncated for column 'p_status' at row 1"}}
{"timestamp": "2024-12-01T15:04:39.061644", "level": "ERROR", "message": "Error in admin_add_property: 1265 (01000): Data truncated for column 'p_status' at row 1", "module": "admin", "function": "admin_add_property", "line": 448, "thread_id": 137674083014464, "environment": "development", "exception": {"type": "DatabaseError", "message": "1265 (01000): Data truncated for column 'p_status' at row 1", "traceback": "Traceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 706, in cmd_query\n    self._cmysql.query(\n_mysql_connector.MySQLInterfaceError: Data truncated for column 'p_status' at row 1\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/routes/admin.py\", line 389, in admin_add_property\n    property_id_result = execute_procedure(\n                         ^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/core/database.py\", line 50, in execute_procedure\n    cursor.callproc(procedure_name, params)\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/cursor_cext.py\", line 566, in callproc\n    result = self._connection.cmd_query(\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/opentelemetry/context_propagation.py\", line 97, in wrapper\n    return method(cnx, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 714, in cmd_query\n    raise get_mysql_exception(\nmysql.connector.errors.DatabaseError: 1265 (01000): Data truncated for column 'p_status' at row 1"}}
{"timestamp": "2024-12-01T15:07:10.848850", "level": "ERROR", "message": "Error fetching properties: 1305 (42000): PROCEDURE real_estate.get_all_properties_with_details does not exist", "module": "main", "function": "index", "line": 36, "thread_id": 137674083014464, "environment": "development"}
{"timestamp": "2024-12-01T15:07:12.356617", "level": "ERROR", "message": "Error fetching properties: 1305 (42000): PROCEDURE real_estate.get_all_properties_with_details does not exist", "module": "main", "function": "index", "line": 36, "thread_id": 137674083014464, "environment": "development"}
{"timestamp": "2024-12-01T15:07:14.730895", "level": "ERROR", "message": "Error fetching properties: 1305 (42000): PROCEDURE real_estate.get_all_properties_with_details does not exist", "module": "main", "function": "index", "line": 36, "thread_id": 137674083014464, "environment": "development"}
{"timestamp": "2024-12-01T15:16:31.027296", "level": "ERROR", "message": "Error in admin_add_property: 1265 (01000): Data truncated for column 'status' at row 1", "module": "admin", "function": "admin_add_property", "line": 450, "thread_id": 134495607871296, "environment": "development", "exception": {"type": "DatabaseError", "message": "1265 (01000): Data truncated for column 'status' at row 1", "traceback": "Traceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 706, in cmd_query\n    self._cmysql.query(\n_mysql_connector.MySQLInterfaceError: Data truncated for column 'status' at row 1\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/routes/admin.py\", line 391, in admin_add_property\n    property_id_result = execute_procedure(\n                         ^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/core/database.py\", line 50, in execute_procedure\n    cursor.callproc(procedure_name, params)\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/cursor_cext.py\", line 566, in callproc\n    result = self._connection.cmd_query(\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/opentelemetry/context_propagation.py\", line 97, in wrapper\n    return method(cnx, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 714, in cmd_query\n    raise get_mysql_exception(\nmysql.connector.errors.DatabaseError: 1265 (01000): Data truncated for column 'status' at row 1"}}
{"timestamp": "2024-12-01T15:16:40.340754", "level": "ERROR", "message": "Error in admin_add_property: 1265 (01000): Data truncated for column 'status' at row 1", "module": "admin", "function": "admin_add_property", "line": 450, "thread_id": 134495607871296, "environment": "development", "exception": {"type": "DatabaseError", "message": "1265 (01000): Data truncated for column 'status' at row 1", "traceback": "Traceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 706, in cmd_query\n    self._cmysql.query(\n_mysql_connector.MySQLInterfaceError: Data truncated for column 'status' at row 1\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/routes/admin.py\", line 391, in admin_add_property\n    property_id_result = execute_procedure(\n                         ^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/core/database.py\", line 50, in execute_procedure\n    cursor.callproc(procedure_name, params)\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/cursor_cext.py\", line 566, in callproc\n    result = self._connection.cmd_query(\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/opentelemetry/context_propagation.py\", line 97, in wrapper\n    return method(cnx, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 714, in cmd_query\n    raise get_mysql_exception(\nmysql.connector.errors.DatabaseError: 1265 (01000): Data truncated for column 'status' at row 1"}}
{"timestamp": "2024-12-01T15:16:41.102144", "level": "ERROR", "message": "Error in admin_add_property: 1265 (01000): Data truncated for column 'status' at row 1", "module": "admin", "function": "admin_add_property", "line": 450, "thread_id": 134495607871296, "environment": "development", "exception": {"type": "DatabaseError", "message": "1265 (01000): Data truncated for column 'status' at row 1", "traceback": "Traceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 706, in cmd_query\n    self._cmysql.query(\n_mysql_connector.MySQLInterfaceError: Data truncated for column 'status' at row 1\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/routes/admin.py\", line 391, in admin_add_property\n    property_id_result = execute_procedure(\n                         ^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/core/database.py\", line 50, in execute_procedure\n    cursor.callproc(procedure_name, params)\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/cursor_cext.py\", line 566, in callproc\n    result = self._connection.cmd_query(\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/opentelemetry/context_propagation.py\", line 97, in wrapper\n    return method(cnx, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 714, in cmd_query\n    raise get_mysql_exception(\nmysql.connector.errors.DatabaseError: 1265 (01000): Data truncated for column 'status' at row 1"}}
{"timestamp": "2024-12-01T15:16:47.116975", "level": "ERROR", "message": "Error in admin_add_property: 1265 (01000): Data truncated for column 'status' at row 1", "module": "admin", "function": "admin_add_property", "line": 450, "thread_id": 134495607871296, "environment": "development", "exception": {"type": "DatabaseError", "message": "1265 (01000): Data truncated for column 'status' at row 1", "traceback": "Traceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 706, in cmd_query\n    self._cmysql.query(\n_mysql_connector.MySQLInterfaceError: Data truncated for column 'status' at row 1\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/routes/admin.py\", line 391, in admin_add_property\n    property_id_result = execute_procedure(\n                         ^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/app/core/database.py\", line 50, in execute_procedure\n    cursor.callproc(procedure_name, params)\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/cursor_cext.py\", line 566, in callproc\n    result = self._connection.cmd_query(\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/opentelemetry/context_propagation.py\", line 97, in wrapper\n    return method(cnx, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/tygilman/School/CS3200/real-estate-htmx-fastapi/.venv/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 714, in cmd_query\n    raise get_mysql_exception(\nmysql.connector.errors.DatabaseError: 1265 (01000): Data truncated for column 'status' at row 1"}}
